content/projects/gsoc/2022/projects/automatic-git-cache-maintenance.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: gsocproject2
+title: "Automatic git cache maintenance on the controller"
+goal: "Automate the maintenance of cached git repositories on the Jenkins controller"
+category: tools
+year: 2022
+sig: platform
+tags:
+- gsoc2022
+- plugins
+status: "Selected"
+student: jmMeessen
```

Comment:
I think this should be hrushi20 or something similar.  I assume that it needs an author entry for Hrushikesh and that is less important right now than completing the blog post so that we can announce.
```suggestion
student: Hrushikesh Rao
```